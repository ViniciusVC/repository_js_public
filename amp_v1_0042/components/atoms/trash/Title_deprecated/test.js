import React from "react";
import { shallow } from "enzyme";
import Component from "./index";

describe("Title >", () => {
  it("deve renderizar com o tema default", () => {
    const wrapper = shallow(<Component />);
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
  it("deve renderizar com a cor primaria", () => {
    const wrapper = shallow(<Component primary={true} />);
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
  it("deve renderizar com a cor secundÃ¡ria", () => {
    const wrapper = shallow(<Component secondary={true} />);
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
});
