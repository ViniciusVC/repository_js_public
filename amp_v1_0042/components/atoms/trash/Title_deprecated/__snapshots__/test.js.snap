// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title > deve renderizar com a cor primaria 1`] = `
<h3
  className="Title sc-bdVaJa ccArrC"
/>
`;

exports[`Title > deve renderizar com a cor secund√°ria 1`] = `
<h3
  className="Title sc-bdVaJa dhoUkX"
/>
`;

exports[`Title > deve renderizar com o tema default 1`] = `
<h3
  className="Title sc-bdVaJa butfLJ"
/>
`;
